src/test/feature/testlib/test_lib.cpp

```diff
@@ -73,6 +74,11 @@ TEST_F(TestCommonLib, TestSqlUtil) {
   util.execSQLFile("testlib/sql/sample.sql", "testlib/ans/sample.ans");
 }
 
+TEST_F(TestCommonLib, TestStringFormat) {
+  auto s1 = hawq::test::stringFormat("welcome %s to apache %s project", "you", "HAWQ");
+  EXPECT_EQ(s1, "welcome you to apache HAWQ project");
```

Comment:
Here welcome is used as transitive verb. Refer webster: http://www.merriam-webster.com/dictionary/welcome